bind: bump to 9.20.10

liuyuf78fk opened the pull request:
**New Features**
- Implemented new `notify-defer` option to defer sending NOTIFY messages for zones.  
  (GL #5259, MR !10465)

**Removed**
- Removed dependency on `libsystemd`, replaced with a manual implementation of `sd_notify`.  
  Reduces attack surface and avoids heavy library dependency.  
  (MR !10454, see also: https://www.openwall.com/lists/oss-security/2024/03/29/4)

**Bug Fixes**
- Fixed issue where deleted secondary zones could still be transferred.  
  (GL #5291, MR !10496)
- Fixed zone refresh failure when reconfiguring during SOA query.  
  (GL #5307, MR !10495)
- Fixed Solaris build: `keystore.c` now includes `isc/dir.h` to define `NAME_MAX`.  
  (GL #5327, MR !10523)
- Set name for all `isc_mem` contexts.  
  (MR !10498)

Tested on:  
Device: Redmi AX6  
Firmware: OpenWrt 24.10

liuyuf78fk committed:
bind: bump to 9.20.10

New notify-defer configuration option
Removed dependency on libsystemd
Fixed zone deletion issue (GL #5291)
Fixed zone refresh bug (GL #5307)

Signed-off-by: Liu Yu <f78fk@live.com>

BKPepe commented on net/bind/Makefile:
```diff
@@ -9,7 +9,7 @@
 PKG_NAME:=bind
-PKG_VERSION:=9.20.9
+PKG_VERSION:=9.20.10
 PKG_RELEASE:=3
```
```suggestion
PKG_RELEASE:=1
```

liuyuf78fk committed:
bind: reset PKG_RELEASE to 1

Set PKG_RELEASE back to 1 as suggested.
This follows OpenWrt packaging convention, which resets PKG_RELEASE to 1
after version bump.

Co-authored-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
Signed-off-by: Liu Yu <f78fk@live.com>

nmeyerhans commented:
Merged with 43681b542062fa8cae309fbea2ec4a627f62d107

pprindeville reviewed (commented):
LGTM
